debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/PostgresToYbValueConverter.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
```

Comment:
Better to drop this header.

Reply:
There is some check that fails during build if we omit this header 😅 
Will look into removing that requirement!